source/pom.xml

```diff
@@ -69,7 +69,6 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <encoding>${source.encoding}</encoding>
```

Comment:
Maven [does not know or care about source.encoding](http://books.sonatype.com/mvnref-book/reference/resource-filtering-sect-properties.html). Maven only knows `project.build.sourceEncoding` and `project.reporting.outputEncoding`. The compiler and resource plugin use these as default for the encoding. The `source.encoding` property is completely unknown to Maven and does not set the encoding on these plugins. Don't get confused by the fact that parameter names of the plugins are also called `encoding`. It's just made up by the POM author. It is if you will a helper variable. The current code is equivalent to this code:

```java
String sourceEncoding = "UTF-8"; // unknown by Maven
String projectBuildSourceEncoding = sourceEncoding; // used by Maven
String projectReportingOutputEncoding = sourceEncoding; // used by Maven
```
you could as well simply use

```java
String projectBuildSourceEncoding = "UTF-8";
String projectReportingOutputEncoding = "UTF-8";
```